services/gemma-local/Makefile

```
@@ -0,0 +1,140 @@
+# Makefile for Gemma Local Service
+
+.PHONY: help install install-dev test test-unit test-integration lint format type-check security-scan clean run run-new docker-build docker-run
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)